platform/security/platform-security-core-api/src/main/java/ddf/security/samlp/SystemCrypto.java

```diff
@@ -82,6 +82,12 @@ private Crypto createCrypto(Properties cryptoProperties) {
     } catch (WSSecurityException e) {
       LOGGER.debug("Error in loading the Crypto object: ", e);
```

Comment:
❓ Should we not throw an IllegalStateException here as well with the caught exception as the cause instead of falling through to retest that it is null and throw it there? I am not suggesting removing that if() below as getInstance() could return null. The fact is that it should probably be moved with the the try instead.